js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@chorusone/solido.js",
+  "version": "1.0.0",
+  "description": "Typescript SDK for interacting with Solido Program",
```

Comment:
Shall we say “for interacting with Lido for Solana (Solido)” to make it more discoverable?

Reply:
Sure we should do this